packages/react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -785,6 +773,32 @@ class Dropdown extends AutoControlledComponent<Extendable<DropdownProps>, Dropdo
     // we don't have event for it, but want to keep the event handling interface, event is empty.
     _.invoke(this.props, 'onSelectedChange', {}, { ...this.props, value })
   }
+
+  private tryFocusTriggerButton = () => {
+    if (!this.props.search) {
+      this.buttonRef.current.focus()
+    }
+  }
+
+  private tryFocusSearchInput = () => {
+    if (this.props.search) {
+      this.inputRef.current.focus()
+    }
+  }
+
+  /**
+   * if there is no value we use the placeholder value
+   * otherwise, for single selection we convert the value with itemToString
+   * and for multiple selection we return empty string, the values are rendered by renderSelectedItems
+   */
+  private getSelectedItemAsString = (value: ShorthandValue): string => {
+    const { itemToString, multiple, placeholder } = this.props
+    return isValueEmpty(value) ? placeholder : multiple ? '' : itemToString(value)
+  }
+}
+
+const isValueEmpty = (value: ShorthandValue): boolean => {
```

Comment:
This function is used once, do we need it? 🤔

Reply:
we don't but I thought it's more readable, removed anyway